method-proposals/PROPOSAL-did-cheqd.md

```diff
@@ -0,0 +1,142 @@
+# DID method standardization proposal: did:cheqd
+
+This is a proposal to include `did:cheqd` in the initial set of DID methods that will be standardized by the DID Methods WG.
+
+## Description
+
+`did:cheqd` is a blockchain-based DID Method that runs atop the cheqd Network. 
+
+- Summary of [DID Method Traits](https://identity.foundation/did-traits/#comparison-of-did-methods)
+
+### Benefits
+
+* Immutable, cryptographically verifiable history of DID Documents and DID Document updates
+* High assurance in integrity of DID Documents and DID Document history, based on delegated Proof of Stake (dPoS) consensus algorithm
+* Key rotation supported with fully indexed, chronologically ordered and searchable history of DID Documents and associated updates
+* Hosted in a decentralized way, eliminating reliance on DNS, web servers or other centralized storage
+* Blockchain-based validation of cryptographic keys stored within the `authentication` section of DID Documents, ensuring that only the controller of those keys has the ability to update or change the DID Document
```

Comment:
Wouldn't necessarily say they are "stored in" that section, as they could be references to own or other DIDDoc. Also, don't think there's any reference to multi-sig capabilities.

Reply:
Amended and added muti-sig section